src/components/media/Media.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+// the outermost element
+.bdl-Media {
+    align-items: flex-start;
+    display: flex;
+}
+
+// the media content, ie, an avatar
+.bdl-Media-figure {
+    align-self: flex-start;
+    flex: 0 0 auto;
+    justify-self: flex-start;
+    margin: 0;
+    padding: 0;
+}
+
+// the main content, e.g. title and message
+.bdl-Media-body {
+    flex: 1 1 auto;
+    margin-left: 10px;
+    // handle wrapping long words in text content without breaking
+    // float position of menu
+    word-break: break-word;
```

Comment:
Were you able to get this to work in IE11?

Reply:
yes

<img width="2003" alt="Screen Shot 2019-07-16 at 4 43 51 PM" src="https://user-images.githubusercontent.com/1571667/61337544-b0171780-a7ea-11e9-8356-e82a1eb7668d.png">